tests/python/contrib/test_pytorch_bridge.py

```diff
@@ -0,0 +1,32 @@
+def pytorch_test():
```

Comment:
The test env do not have torch in its env, do a try import torch and skip test if torch is not available